spec/import_spec.rb

```diff
@@ -44,6 +44,41 @@ def upload_file!(name, ext='csv')
     find_button('Import').click
   end
 
+  context "posts index" do
+    before do
+        Author.create!(name: 'John', last_name: 'Doe')
+        Author.create!(name: 'Jane', last_name: 'Roe')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.